### PR TITLE
Added an Article

### PR DESCRIPTION
**Pull request by Timtech4u**

### CODE REVIEW

**Comment by glaforge** [comment]
I noticed plenty of broken screenshots in the article.

**Comment by Timtech4u** [comment]
Hi @glaforge  I just modified the link.

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 ### Articles
 
+* [Google Cloud Run — Deploying Containerized Applications to a Serverless Environment ⚡](https://medium.com/@timtech4u/deploy-serverless-container-google-cloud-run-68d716af7716)
```
Comment (steren) [inline README.md]: Thanks for the suggestion. This seems to be more a tutorial than an article, would you mind moving it to the next section?

Comment (Timtech4u) [reply README.md]: Sure, will do now.
Adding another article also.

**Comment by steren** [comment]
Thanks. I'll merge and then probably move them around

**Pull request merged**